Scripts/Runtime/Entities/Lifecycle/Migration/MigrationUtil.cs

```diff
@@ -113,23 +116,43 @@ public static void RegisterTypeForEntityPatching(Type type)
                 new TypeOffsetInfo(
                     entityOffsetStartIndex,
                     s_EntityOffsetList.Length));
-            
+
             //The size of the underlying data could have changed such that we re-allocated the memory, so we'll update
             //our shared statics
             UpdateSharedStatics();
         }
 
+        private static void ScanForCollections(string parentPathString, Type type)
```

Comment:
Should this be a safety only operation? I don't think this would come up uniquely in release